docs/pages/management/guides/docker.mdx

```diff
@@ -87,7 +87,7 @@ $ docker run --hostname localhost --rm \
 $ docker run --hostname localhost --name teleport \
   -v ~/teleport/config:/etc/teleport \
   -v ~/teleport/data:/var/lib/teleport \
-  -p 3023:3023 -p 3025:3025 -p 3080:3080 \
+  -p 3023:3023 -p 3025:3025 -p 443:443 \
```

Comment:
I'd revert this unless you actually tested the guide with this config.

I suspect you probably want `-p 443:3080`, but that may require root.

Reply:
As a rule of thumb we try to avoid listening on < 1024 ports in docker container because it requires root or some other kind of privilege not set by default. We are working to stop running containers as root so this would break.

As the docker daemon is root, it should be possible to redirect the host's 443 port to the container 3080 and still provide a unified client experience as Zac suggests.